core/query/mixingQuery.ml

```diff
@@ -208,10 +208,11 @@ let rec reduce_where_then (c, t) =
   | _ ->
     Q.If (c, t, Q.Concat [])
 
-let reduce_for_body (gs, os, body) =
+let rec reduce_for_body (gs, os, body) =
+  let rb = reduce_for_body in
   match body with
     (* | Concat []                 -> body *)
```

Comment:
Is it worth removing this commented-out line now that Concat is handled uniformly by a new case?